### PR TITLE
HP-1592: Correct layout for asking read rights for user data

### PR DESCRIPTION
**Pull request by NikoHelle**
Updated the skeleton layout, added new translations and adjusted styles

### CODE REVIEW

File: helsinki/login/service-connection-confirm-form.ftl [hunk 1]
```diff
@@ -1,21 +1,20 @@
...
+      <p>${kcSanitize(msg("serviceDataUsageListTitleOldUser",serviceName))}</p>
+      <ul class="checked-list content-before-buttons">
+        <#list serviceAllowedData as item>
+          <li>${item}</li>
+        </#list>
+      </ul>
+      <form id="kc-service-connection-confirm-form" action="${url.loginAction}" method="post">
+          <div class="wide-buttons">
+              <button class="${properties.kcButtonClass!} ${properties.kcButtonPrimaryClass!} ${properties.kcButtonLargeClass!}" id="kc-accept" type="submit" name="response" value="accept">${msg("profileAcceptButtonText")}</button>
+              <button class="${properties.kcButtonClass!} ${properties.kcButtonDefaultClass!} ${properties.kcButtonLargeClass!}" id="kc-decline" type="submit" name="response" value="decline">${msg("profileDeclineButtonText")}</button>
```
Comment (akikoskinen) [inline helsinki/login/service-connection-confirm-form.ftl:15]: Some design has texts "Anna lupa" and "Älä anna lupaa" in the buttons. Is that part of the design obsolete or what should the button texts be?

Comment (NikoHelle) [reply helsinki/login/service-connection-confirm-form.ftl:15]: Design is an overwrite on the Tunnistamo version of the permission dialog. "Jatka" and "Peruuta" are used in about all views in Keycloak and approved texts in the buttons.

Comment (akikoskinen) [reply helsinki/login/service-connection-confirm-form.ftl:15]: I take that as to mean that we are not obeying the design to the point, but use the texts that the template currently has :smirk:

**Review by akikoskinen — APPROVED** [review]

File: helsinki/login/service-connection-confirm-form.ftl [hunk 2]
```diff
@@ -1,21 +1,20 @@
...
-        </div>
-        <form id="kc-service-connection-confirm-form" action="${url.loginAction}" method="post">
-            <div>
-                <button id="kc-accept" type="submit" name="response" value="accept">${msg("profileAcceptButtonText")}</button>
-                <button id="kc-decline" type="submit" name="response" value="decline">${msg("profileDeclineButtonText")}</button>
-            </div>
-        </form>
+      <p>${kcSanitize(msg("serviceDataUsageListTitleOldUser",serviceName))}</p>
+      <ul class="checked-list content-before-buttons">
+        <#list serviceAllowedData as item>
```
Comment (akikoskinen) [inline helsinki/login/service-connection-confirm-form.ftl:8]: `serviceAllowedData` must be renamed to `serviceAllowedDataFields`.

**Pull request merged**